mon-pix/tests/integration/components/authentication/password-reset-demand-form-test.gjs

```diff
@@ -89,11 +103,26 @@ module('Integration | Component | Authentication | password-reset-demand-form',
 
         // then
         assert.dom(screen.queryByRole('alert')).doesNotExist();
+
+        assert
+          .dom(
+            screen.queryByRole('heading', {
+              name: t('components.authentication.password-reset-demand-received-info.heading'),
+            }),
+          )
+          .exists();
+
+        const tryAgainLink = await screen.queryByRole('link', {
+          name: t('components.authentication.password-reset-demand-received-info.try-again'),
+        });
+        assert.dom(tryAgainLink).exists();
+        assert.strictEqual(tryAgainLink.getAttribute('href'), '/mot-de-passe-oublie');
       });
     });
 
+    // TODO: This test module will need to be removed when the API doesn't leak anymore that an account doesn't exist with a "404 Not Found".
     module('when there is no corresponding user account', function () {
-      test('it displays an "account not found" error message', async function (assert) {
+      test('it also doesn’t display any error message and displays a "password reset demand received" info', async function (assert) {
```

Comment:
Je suggère de mettre en avant l'intention principale du test (et enlever `also` car les tests sont indépendant entre eux):
```suggestion
      test('it displays a "password reset demand received" info (without any error message to avoid email enumeration)', async function (assert) {
```

Reply:
Très bonne idée. Fait.